export_doc.py

```diff
@@ -1,9 +1,12 @@
+"""
+Export doc
+"""
 import json
 import datetime
 from pathlib import Path
 import argparse
 
-
+# NOTE Redefining name 'notebook_path' from outer scope (line 39)pylint(redefined-outer-name)
```

Comment:
Please keep any linting related corrections to the pylintrc, as I don't want these comments to worsen readability.

Reply:
Alright, I am changing notebook_path to given_notebook_path as lint workaround here.